### DISCUSSION TITLE
Browser-compatible version to npm

### DISCUSSION THREAD

**Discussion by furedal** [exceljs/exceljs#2539] [Q&A]
As I understand it, there is a version that should be compatible to be run on browser am I right?

I want to try out this project in my React-application but the current version seem to require node.
Could the browser-compatible version be published using npm?

**Comment by zl810881283** [comment]
+1

**Comment by zl810881283** [comment]
已经收到您的来信，我会尽快回复。

**Comment by Siemienik** [comment]
https://github.com/exceljs/exceljs?tab=readme-ov-file#browserify
